tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -268,3 +268,10 @@ user deletes editable accordion section content block
     ${block}=  get child element  ${section}  css:[data-testid="editableSectionBlock"]:nth-of-type(${block_num})
     user clicks button  Remove block  ${block}
     user clicks button  Confirm
+
+user gets meta guidance accordion text element
+    [Arguments]   ${accordion_heading}
+    ${accordion}=  user gets accordion section content element  ${accordion_heading}
+    user waits until parent contains element  ${accordion}   xpath://*[@data-testid="Content"]//*[contains(@class,"ck-content")]
```

Comment:
Checking based on `ck-content` is likely to be brittle and also not descriptive. Use `role="textbox"` instead.